tests/repository/offer_queries_test.py

```diff
@@ -395,6 +395,76 @@ def test_search_does_not_return_duplicate(self, app):
         assert len(search_result_offers) == len(set(search_result_offers))
 
 
+    @clean_database
+    def test_search_does_not_return_not_active_offer(self, app):
```

Comment:
```suggestion
    def test_should_not_return_a_deactivated_offer_when_searching(self, app):
```
Je vais en parler lors de notre réunion tech mais de façon générale, je trouve qu'il est plus lisible d'écrire un test `shoud return [...] when [...]` et l'inverse en front. Ou du moins, d'avoir toujours la même structure d'intitulé.
Je te laisse revoir les autres intitulés.

Reply:
:+1: